nana/modules/admins.py

```diff
@@ -626,20 +572,18 @@ async def lock_permission(client, message):
                     can_pin_messages=pin,
                 ),
             )
-
-            await msg(message, text=f"`Locked {perm} for this chat!`")
+            await edrep(message, text=tld('lock_chat').format(perm))
             await asyncio.sleep(5)
             await message.delete()
-
         except Exception as e:
             print(e)
-            await msg(message, text="Something Happened Please Check logs.")
+            await message.delete()
             return
     else:
         await message.delete()
 
 
-@app.on_message(Filters.user(AdminSettings) & Filters.command("unlock", Command))
+@app.on_message(filters.user(AdminSettings) & filters.command("unlock", Command))
```

Comment:
Codacy found an issue: [unlock_permission is too complex (20) (MC0001)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)